### PR TITLE
feat: Display off days banner if donation banner was hidden

### PR DESCRIPTION
**Pull request by Valimp**
### What

We have to integrate off days 2024 banner, but we want to display it only if donnation banner was closed

### CODE REVIEW

**Comment by Valimp** [comment]
TO DO: 
- Manipulate DOM to reload banner when you close the donation banner
- Change images and links depending of lang (FR, EN)

File: templates/web/common/includes/off_days_2024_banner.tt.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    }
+
+    if (getBannerCookie('off_donation_banner_2023_a') !== '') {
+        console.log('cookie found');
+        offDaysBannerID.style.display = 'flex';
+    } else {
+        console.log('cookie not found');
+        offDaysBannerID.style.display = 'none';
+    }
+</script>
```
Comment (Valimp) [inline templates/web/common/includes/off_days_2024_banner.tt.html:31]: This code works, but i need to reload page to display off days banner.
Do you have an idea to reload only banners ? 
@alexgarel

**Comment by Valimp** [comment]
Strings a add in en.po and fr.po doesn't work, i need to do something more to make templates works ?

File: templates/web/common/includes/off_days_2024_banner.tt.html (hunk 1 above)
Comment (alexgarel) [reply templates/web/common/includes/off_days_2024_banner.tt.html:31]: As discussed, this is not a big deal, banner can be displayed on next page, the invitation is really aimed at frequent contributors.

**Comment by alexgarel** [comment]
> Strings a add in en.po and fr.po doesn't work, i need to do something more to make templates works ?

make build_lang

**Comment by Valimp** [comment]
Laptop screen EN
Capture d’écran 2024-07-23 à 16 19 59

Phone screen EN
Capture d’écran 2024-07-23 à 16 20 35

Laptop screen FR
Capture d’écran 2024-07-23 à 16 21 04

Phone screen FR
Capture d’écran 2024-07-23 à 16 21 35

Links to register page depends on lang

**Review by stephanegigandet — APPROVED** [review]
Thanks

**Pull request merged**